app/views/admin/calendars/_form.html.erb

```diff
@@ -23,7 +23,7 @@
                           hint: 'What group organises these events?' %>
 
         <%= f.input :name, hint: 'A simple description to help rememember what this calendar is' %>
-        <%= f.input :source, label: 'URL', placeholder: 'https://your-domain.com/events.ics', hint: "The source URL for your calendar feed. See the #{link_to 'PlaceCal Handbook', 'https://handbook.placecal.org/admins/supported-sources.html'} if you need help.".html_safe %>
+        <%= f.input :source, label: 'URL', placeholder: 'https://your-domain.com/events.ics', hint: "The source URL for your calendar feed. See the #{link_to 'PlaceCal Handbook', 'https://gfsc.notion.site/Supported-Event-Sources-603f67d632c24aebbb32b9be7edebfa5'} if you need help.".html_safe %>
```

Comment:
i think we should link to https://handbook.placecal.org/admin-handbook instead, this is way too brittle.

Reply:
I'll change it but I find it frustrating when a link to a resource doesn't actually lead to the bit you need.